### DISCUSSION TITLE
Scratch Addons YouTube Channel Idea Spam

### DISCUSSION THREAD

**Discussion by Cool-Scratcher** [ScratchAddons/ScratchAddons#4564] [Discussion]
Hello everyone, video guy here. I'm trying to compile a list of video ideas related to the SA YouTube channel into a todo list. I have this on GitHub (even though I am obsessed with Discord) because 1) I'm on my school computer rn and 2) Discord is not very organized for discussions. Also not everyone has Discord.

There's [a Google doc](https://docs.google.com/document/d/19m8gfnELEGw-oFT42Pp_1iAu4kmkA2TCgF7fUEQ0mpQ/edit?usp=sharing) I made. You can take a look there and suggest new ideas. New ideas should be discussed in-depth here.

I already put some existing ideas up for debate there. Take a look and share your thoughts. I hope I'm doing this right lmao

**Comment by Norbiros** [comment]
Some of my (maybe) stupid ideas:
- Localization BTS
    I think it should contain 2 parts: developers and translators.
    In translator part:
    - How join our team on transifex
    - Example of translating some strings
    - Maybe some Tips?
   In developers part:
     - How to create translations?
     - How to add new "Developer Comments" to help translators...
- Scratch Addons Shorts
    Maybe we can show [look at *Scratch things*], not just SA things (but we can show things like new addons, changes etc.)
- How to make own addon?
    When I was starting with SA, it was very confusing. Maybe video with step by step tutorial? After that we can create new parts (things from docs)
- More memes?
   More things like this: https://www.youtube.com/watch?v=zC-CzFb8wdM
- Scratch things
   The best projects showcase, programming tips, easter eggs showcase...

**Reply by cobaltt7** [reply]
> Localization BTS

i like the idea of a "how to translate" video
> Scratch Addons Shorts

agreed
> How to make own addon?

we kinda started this but cs and i both think that it's not turning out well so the idea was ditched
> More memes?

yes

> Scratch things

maybe

**Reply by Cool-Scratcher** [reply]
I don't like the memes idea tbh, that was more of when the channel was starting out and jeffalo was bored or something idk

**Comment by shock59** [comment]
I like the idea of some general Scratch things. Perhaps even some tutorials but more general tips rather than full projects like griffpatch creates. We could involve the community by asking for everyone's favourite tips (combination of general Scratch and SA tips) on the Discord and compiling them all into one longer video. Shorts of smaller tips would also be good but I don't know how many people who would be interested in Scratch watch Shorts. Some other community events would be cool too, but I'm not sure what kind of events those would be.

**Reply by Cool-Scratcher** [reply]
> I like the idea of some general Scratch things

I don't know how much that would relate to SA. General tips would work, but imo only if they're SA-related.

**Reply by shock59** [reply]
I mean jeffalo wanted to make Scratch tutorials in combination with SA content to get the channel more popular

**Comment by Cool-Scratcher** [comment]
What about some sort of "100 facts about SA" video? Maybe as one of the celebrations for the 10K users event, or just as random video uploaded to the SA channel. It would take a LONG time to make, though

**Reply by Norbiros** [reply]
But this facts cannot be boring. If we gonna have facts like:
- scratch addons is extension
- scratch addons is avaible on chrome
- scratch addons is avaible on firefox
- scratch addons' source code is on github

It is not gonna be fun. IMAO it is better to make for example 50 interesting facts....

**Reply by Cool-Scratcher** [reply]
> IMAO it is better to make for example 50 interesting facts....

Well yeah they'd need to be reasonably interesting. Trivia of sorts maybe? Something like "Scratch Addons was based off a few old extensions" or "The only major SA contributor who doesn't have Discord is mxmou" (I don't know about that last one for sure though)

- Mega Scratch Userscript existed and was an inspiration for SA
- Griffpatch is not the only developer, and neither is WL or GarboMuffin or me (you'll be surprised how many people think that lmao)
- The SA website has gone through two versions (has it? afaik)
- SA 2.0 is going to be a massive release
- Vue is literally a meme within the dev server
- SA contributors are international (proceed to list all the countries)
- Videos take a long time to make
- Video translation/dubbing coming soon. Maybe. Eventually.

 Basically just fun facts and bts things that not many people know.

**Reply by shock59** [reply]
Some could also be about the SA community since most people aren't in the discord but I think the majority should be about the extension

**Comment by Hans5958** [comment]
Just dropping here to say to also get something for [the blog](https://scratchaddons.com/blog) lol

**Reply by Cool-Scratcher** [reply]
Blog versions of videos????

**Reply by WorldLanguages** [reply]
Possibly.
What I would like to see are blog posts showcasing how to make the best out of every addon, specially the most complete and complex ones, such as debugger. If these blog posts ever get a video version, they could be added inside the post as an embed,

**Reply by Hans5958** [reply]
Something like Addon of the week/month would be good, but "Addon Highlights" or "Addon 101" is good I presume?

**Comment by Cool-Scratcher** [comment]
I updated the google document with new ideas and a categorizing system 😎

**Reply by WorldLanguages** [reply]
epic!

**Reply by shock59** [reply]
Great

**Comment by shock59** [comment]
For large videos such as 100 facts or every addon explained, having multiple hosts/narrators (i.e. not just CoolScratcher) in the same video could add some variety and keep it interesting if it's long
